homeassistant/components/lyric/sensor.py

```diff
@@ -74,6 +74,22 @@ def get_setpoint_status(status: str, time: str) -> str | None:
 
     for location in coordinator.data.locations:
         for device in location.devices:
+            if device.indoorTemperature:
+                entities.append(
+                    LyricSensor(
+                        coordinator,
+                        LyricSensorEntityDescription(
+                            key=f"{device.macID}_indoor_temperature",
+                            name="Indoor Temperature",
+                            device_class=SensorDeviceClass.TEMPERATURE,
+                            state_class=SensorStateClass.MEASUREMENT,
+                            native_unit_of_measurement=hass.config.units.temperature_unit,
+                            value=lambda device: device.indoorTemperature,
+                        ),
+                        location,
+                        device,
+                    )
+                )
             if device.indoorTemperature:
```

Comment:
Is this not the same as the code below?

Reply:
I think you might have missed changing temperature to humidity instead here

Reply:
Thank you. I've put the correct code in.

Reply:
hi @timmo001 I've corrected the incorrect upload. Would you be able to look it over?